build-system/release-tagger/create-release.js

```diff
@@ -103,23 +149,40 @@ function _createSections(prs) {
 
 /**
  * Create body for GitHub release
+ * @param {string} head
  * @param {string} base
  * @param {Array<GraphQlQueryResponseData>} prs
  * @return {string}
  */
-function _createBody(base, prs) {
-  const rawNotes = prs.map(_formatPullRequestLine);
-  const sections = _createSections(prs);
+function _createBody(head, base, prs) {
+  const bento = _createPackageSections(prs);
+  const components = _createComponentSections(prs);
   const template = dedent`\
-      #### *Baseline release: [${base}]\
-      (https://github.com/ampproject/amphtml/releases/${base})*
-
-      #### Raw notes
-      ${rawNotes.join('\n')}
-
-      #### Breakdown by component
-      ${sections.join('')}\
-  `;
+     <h2>Changelog</h2>
+     <p>
+     <a href="https://github.com/ampproject/amphtml/compare/${base}...${head}">
+     <code>${base}...${head}</code>
+     </a>
+     </p>
+ 
+     ${Object.entries(bento)
+       .map(
+         ([major, {packages, unchanged}]) => dedent`\
+         <h2>npm packages @ ${getSemver(major, head)}</h2>
+         ${Object.entries(packages)
+           .map(
+             ([extension, prs]) =>
+               `${extension}\n<ul>${[...prs].join('\n')}</ul>`
+           )
+           .join('\n')}
+   
+         Packages not changed: <i>${[...unchanged].join(', ')}</i>`
```

Comment:
Formatting bikeshed for readability: Think it's worth bolding the extension name on line 175, and the `Package not changed:` header on line 179?